logstash_formatter/__init__.py

```diff
@@ -89,9 +89,15 @@ def format(self, record):
                 fields['exception'] = formatted
             fields.pop('exc_info')
 
-        if 'exc_text' in fields and not fields['exc_text']:
+        if 'exc_text' in fields:
```

Comment:
it becomes a duplicate because exception contains all the text from exc_text so marc asked for it removed. where as this only removes it if the key is there but there is a falsey value